central/graphql/handler/handler_benchmark_test.go

```diff
@@ -0,0 +1,221 @@
+//go:build sql_integration
+
+package handler
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"reflect"
+	"testing"
+
+	"github.com/graph-gophers/graphql-go"
+	"github.com/stackrox/rox/central/graphql/resolvers"
+	"github.com/stackrox/rox/central/graphql/resolvers/loaders"
+	"github.com/stackrox/rox/central/testutils"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/grpc/authz/allow"
+	"github.com/stackrox/rox/pkg/sac"
+)
+
+const (
+	deploymentsQuery = `query listDeployments($query:String){
+	deployments(query: $query) {
+		id
+		name
+		type
+		namespace
+		namespaceId
+		stateTimestamp
+	}
+}`
+
+	imagesQuery = `query listImages($query:String){
+	images(query: $query) {
+		id
+		name {
+			registry
+			remote
+			tag
+			fullName
+		}
+		# scan is required to ensure the full image is pulled
+		# and not just the associated metadata.
+		scan {
+			operatingSystem
+			scanTime
+		}
+	}
+}`
+
+	deploymentsWithImagesQuery = `query listDeploymentsWithImages($query: String) {
+	deployments(query: $query) {
+		id
+		name
+		type
+		namespace
+		namespaceId
+		stateTimestamp
+	}
+	# For the purpose of the test, the plan is to pull the deployments
+	# and images filtered by namespace. The query resolution for images
+	# relies on the relation to the deployment table, so the behaviour
+	# should be similar to the output of "/v1/export/vuln-mgmt/workloads/"
+	images(query: $query) {
+		id
+		name {
+			registry
+			remote
+			tag
+			fullName
+		}
+		# scan is required to ensure the full image is pulled
+		# and not just the associated metadata.
+		scan {
+			operatingSystem
+			scanTime
+		}
+	}
+}`
+)
+
+func BenchmarkDeploymentExport(b *testing.B) {
+	_ = b
+	testHelper := &testutils.ExportServicePostgresTestHelper{}
+	err := testHelper.SetupTest(b)
+	if err != nil {
+		b.Error(err)
+	}
+	defer testHelper.TearDownTest(b)
```

Comment:
I'm not sure if this is better
```suggestion
	b.Cleanup(func() { testHelper.TearDownTest(b) })
```